### PR TITLE
revert kitchensink

### PR DESCRIPTION
**Pull request by peterbe**

### CODE REVIEW

File: files/en-us/mdn/kitchensink/index.html [hunk 1]
```diff
@@ -0,0 +1,361 @@
+---
+title: The MDN Content Kitchensink
+slug: MDN/Kitchensink
+tags:
+  - Kitchensink
+  - MDN Meta
+---
+<div>{{MDNSidebar}}</div>
+
```
Comment (chrisdavidmills) [inline files/en-us/mdn/kitchensink/index.html:9]: Maybe add a warning at the top of the page (in a warning box) along the lines of "Warning: This page is used in a number of platform tests. Please do not delete or move this page."

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Just one small comment.

File: files/en-us/mdn/kitchensink/index.html (hunk 1 above)
Comment (peterbe) [reply files/en-us/mdn/kitchensink/index.html:9]: Sure!
Screen Shot 2021-01-06 at 9 57 11 AM

**Review by chrisdavidmills — APPROVED** [review]
r+ !

**Pull request merged**